psi4/src/psi4/fnocc/df_ccsd.cc

```diff
@@ -418,6 +422,9 @@ PsiReturnType DFCoupledCluster::CCSDIterations() {
   outfile->Printf("        D1 diagnostic:                  %20.12lf\n",sqrt(eigval->pointer()[0]));
   outfile->Printf("\n");
 
+  // add D1 diagnostic to globals
+  Process::environment.globals["CC D1 DIAGNOSTIC"] = sqrt(eigval->pointer()[0]);
+
```

Comment:
@dgasmith brought up a good point in discussion of #711 that we might want to hold of on duplicating `P::e.globals` into `wfn.variables` until we just do an entire module at the same time so we don't have a half done transition.

To keep things consistent, maybe remove the `set_variable` lines or if it isn't too complicated go through and duplicate all of them up to you @edeprince3.